cincinnati/src/plugins/mod.rs

```diff
@@ -4,7 +4,9 @@
 pub mod external;
 pub mod interface;
 pub mod internal;
+mod registry;
```

Comment:
we lost a note on the way discussing this name. I suggest one of:
* catalog
* index

Reply:
I'm +1 on catalog, will rename on next rebase.